src/vos/ilog.h

```diff
@@ -174,7 +174,7 @@ struct ilog_entry {
 	int32_t		ie_status;
 };
 
-#define ILOG_PRIV_SIZE 240
+#define ILOG_PRIV_SIZE 392
```

Comment:
why did the size change here?

Reply:
The umem_instance is enlarged, it's included in ilog priv through ilog_context.ic_umem.